migrations/1534799072630_assets-rename-asset-id-to-token-id.js

```diff
@@ -0,0 +1,7 @@
+import { Parcel } from '../src/Parcel'
+import { Estate } from '../src/Estate'
+
+exports.up = pgm => {
```

Comment:
It would be great to have the `.down` just in case.

it is useful also if we need to switch branches until we have it on master